base:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Effort Controllers ---------------------------------------
  wheel_left_effort_controller:
    type: effort_controllers/JointEffortController
    joint: wheel_left_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  wheel_right_effort_controller:
    type: effort_controllers/JointEffortController
    joint: wheel_right_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
